templates/data/api-gateway-manager-build.yaml

```diff
@@ -62,22 +62,102 @@ templates:
                     - "image_buildpack-golang" # takes latest golang image
                   local:
                     - job_default
+              - jobConfig: # builds an OCI image with api-gateway module from mod-dev branch
+                  name: post-api-gateway-module-build
+                  annotations:
+                    owner: "goat"
+                    description: "API Gateway Manager template image build on mod-dev branch"
+                  always_run: true
+                  optional: true
```

Comment:
Do we want optional to be set to true here?

Reply:
In the next pushed version.